src/components/input-panel/spec-editor/renderer.js

```diff
@@ -107,7 +106,7 @@ export default class Editor extends React.Component {
             lineNumbersMinChars: 4
           }}
           defaultValue={this.props.value}
-          onChange={debounce(this.handleEditorChange, 500).bind(this)}
+          onChange={debounce(this.handleEditorChange, 700).bind(this)}
```

Comment:
It seems like a hack, but the error goes away after the debounce time increases.